src/index.html

```diff
@@ -5,9 +5,395 @@
+            </p>
+          </div>
+
+          <div class="testimonials__person person">
+            <div class="person__img-position">
+              <img
+                src="./images/people/Ellipse 3.png"
+                alt="Max Henry"
+                class="person__photo"
+              >
+            </div>
+            <h4 class="person__name">Max Henry</h4>
+            <span class="person__position">Chief Design Officer</span>
+            <p class="person__title">When my lovely valley of steam rises and half-day sun is above an impermeable</p>
+          </div>
+
+        </div>
+      </div>
+    </div>
+
+    <div class="contact" id="contact">
+      <div class="contact__container container">
+        <h2 class="contact__heading block__heading">Contact us</h2>
+        <div class="contact__main-block">
+          <div class="contact__data">
+            <div class="contact__phone-email">
+              <h6 class="contact__small-heading">Phone</h6>
+              <span>
+                <a href="tel:+1 234 5555-55-55" class="contact__text">+1 234 5555-55-55</a>
+              </span>
+              <h6 class="contact__small-heading">Email</h6>
+              <span>
+                <a href="mailto:hello@miami.com" class="contact__text">hello@miami.com</a>
+              </span>
+            </div>
+            <div class="contact__adress">
+              <h6 class="contact__small-heading">Address</h6>
+              <p >
+                <a
+                  href="https://www.google.com/maps/search/?api=1&query=400+first+ave.+n.+suite+700+Minneapolis,+MN+55401"
+                  target="_blank"
+                  rel="noopener"
+                  class="contact__text">
+                  400 first ave. n.<br>suite 700<br>Minneapolis, MN <br>55401
+                </a>
+              </p>
+            </div>
+          </div>
+
+          <div>
+            <form action="" class="contact__form">
+              <div class="form__name">
+                <input
+                  type="text"
+                  name="name"
+                  id="name contact"
+                  required
+                  autocomplete="off"
+                  placeholder="Name"
+                  class="form"
+                >
+              </div>
+
+              <div class="form__email contact__email">
+                <input
+                  type="email"
+                  name="email"
+                  id="email contact"
+                  required
+                  placeholder="Email"
+                  class="form"
+                >
+              </div>
+
+              <div class="form__message">
+                <textarea
+                  placeholder="Message"
+                  id="message contact"
+                  required
+                  class="textarea"
+                >
+                </textarea>
```

Comment:
ось так placeholder має з'явитись
```suggestion
                <textarea
                  placeholder="Message"
                  id="message contact"
                  required
                  class="textarea"
               ></textarea>
```